lib/x11regressiontest.pm

```diff
@@ -481,9 +481,20 @@ sub exit_firefox() {
 }
 
 sub start_gnome_settings {
-    send_key "super";
-    wait_still_screen;
-    type_string "settings";
+    my $is_sle_sp1 = check_var('VERSION', '12-SP1');
+    my $workaround_repetitions = 3;
+
+    do {
+        if ($is_sle_sp1) {
+            send_key "super";    # if launcher open, close (search string will also be removed).
```

Comment:
Yes, I first tried pressing 'esc' twice but it didn't worked. Somehow pressing the same key too fast is sometimes lost.
So I am trying to use two different keys. I know that pressing 'super' will close the launcher and removed the typed string. I could simply press super, but sometimes the bug makes the launcher to close, so it pressing 'super' will opening it again.